redislock.go

```diff
@@ -176,14 +206,18 @@ func (l *Lock) TTL(ctx context.Context) (time.Duration, error) {
 // Refresh extends the lock with a new TTL.
```

Comment:
You need to update the comment to explain what is happening when multiple locks are held. What if you hold two locks and one can be extended and the other isn't? This is not a trivial problem.

Reply:
How would one lock be extended and not the other ?
Given the atomicity of the script, either both lock exists and you can refresh them or at least one is missing and none are refreshed.